inst/pages/21_microbiome_community.qmd

```diff
@@ -199,7 +199,7 @@ sample_data <- data.frame(clusters = sample_clusters)
 # Order data so that it's same as in phylo tree
 sample_data <- sample_data[samples_ordered, , drop = FALSE] 
 
-# Order data based on 
+# Order data based on the phlyo tree
```

Comment:
typo

Reply:
Sorry, I did not understand you here, do you mean i should the remove the added part?

Reply:
phlyo is typo

Reply:
Oh no sorry I was confused, fixed it.